plots/scripts/7-plot_plier_pathways.R

```diff
@@ -45,15 +45,27 @@ output_filename <- file.path(output_directory,
                              str_c(file_identifier,
                                    "_PLIER_jaccard.pdf"))
 
+# sample size levels
+sample_size_levels = c("Single Platform\n(half sample size)",
```

Comment:
I think this is just assignment, so
```suggestion
sample_size_levels <- c("Single Platform\n(half sample size)",
```